package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "coveo-search-ui",
-  "version": "2.0.606",
+  "version": "2.0.625",
```

Comment:
Looks like Github's comparison is broken. If you look at JSUI-2949, you can see that it's not 2.0.606.